content/en/logs/indexes/_index.md

```diff
@@ -78,6 +78,37 @@ Update or remove this quota at any time when editing the Index:
 
 **Note**: Indexes daily quotas reset automatically at 2:00pm UTC (4:00pm CET, 10:00am EDT, 7:00am PDT).
 
+### Attribute based sampling
```

Comment:
What about moving those two independent section into the main exclusion filter right after the definition?

Even as #### if you want to be able to link directly to it but here it looks a bit far away from the exclusion filter definition.